osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointPiece.cs

```diff
@@ -98,6 +111,9 @@ protected override bool OnDrag(DragEvent e)
 
             slider.Path = new SliderPath(slider.Path.Type, newControlPoints);
 
+            sliderUpdateDebounce?.Cancel();
+            sliderUpdateDebounce = Schedule(() => slider.ApplyDefaults(beatmap.Value.Beatmap.ControlPointInfo, beatmap.Value.BeatmapInfo.BaseDifficulty));
```

Comment:
what is the debounce and schedule here for?

Reply:
I requested this be added here, to stop potentially 1000s of `ApplyDefaults` being triggered.

Reply:
Can more than one drag events actually fire between a single update frame?

Reply:
Yes, this is the reason various things like music controller seeks are debounced.